presto-jmx/src/test/java/io/prestosql/plugin/jmx/TestJmxTableHandle.java

```diff
@@ -48,23 +56,26 @@ public void testEquivalence()
         List<JmxColumnHandle> singleColumn = ImmutableList.of(COLUMNS.get(0));
         EquivalenceTester.equivalenceTester()
                 .addEquivalentGroup(
-                        new JmxTableHandle(SCHEMA_TABLE_NAME, ImmutableList.of("name"), COLUMNS, true),
-                        new JmxTableHandle(SCHEMA_TABLE_NAME, ImmutableList.of("name"), COLUMNS, true))
+                        new JmxTableHandle(SCHEMA_TABLE_NAME, ImmutableList.of("name"), COLUMNS, true, TupleDomain.all()),
+                        new JmxTableHandle(SCHEMA_TABLE_NAME, ImmutableList.of("name"), COLUMNS, true, TupleDomain.all()))
+                .addEquivalentGroup(
+                        new JmxTableHandle(SCHEMA_TABLE_NAME, ImmutableList.of("name"), COLUMNS, false, TupleDomain.all()),
+                        new JmxTableHandle(SCHEMA_TABLE_NAME, ImmutableList.of("name"), COLUMNS, false, TupleDomain.all()))
                 .addEquivalentGroup(
-                        new JmxTableHandle(SCHEMA_TABLE_NAME, ImmutableList.of("name"), COLUMNS, false),
-                        new JmxTableHandle(SCHEMA_TABLE_NAME, ImmutableList.of("name"), COLUMNS, false))
+                        new JmxTableHandle(SCHEMA_TABLE_NAME, ImmutableList.of("nameX"), COLUMNS, true, TupleDomain.all()),
```

Comment:
Why the name change here?

Reply:
This is diff magic. I didn't change the name, just added new arguments. "name" patterns are coverd by preceding 2 test cases.